server.js

```diff
@@ -28,6 +28,7 @@ var parseTweets = function(body) {
     });
   });
 
+  tweets.pop();
```

Comment:
@davidlimcheng although this works for this particular usecase, it might be good to approach this a bit different. Instead, can we wrap the `tweets.push({` method in an `if` clause?

e.g. only push things on the array that have `text`.